docs/Documentation/Scripting/Building-Blocks/Objectives-List.md

```diff
@@ -600,3 +600,21 @@ by what players type in chat which is only useful when you're also using the `va
     ```YAML
     variable
     ```
+
+# Variables in Objectives
+
+Warning: Use with caution!
+
+Information: Variable means here the variable with `%%`, not global variables with `$$`!
+Global variables are absolutely safe and will result in an error on reload if used wrong.
+
+Instead of hardcoded numbers you can use variables as amount in Objectives.
+Note that the value of the variable at the moment of assigning the objective is used and will then be stored per active objective independent.
+To assign a new value you would have to delete the old objective and start a new one,
+discards all process.
+Also, when an invalid amount for the objective is given a default value of `1` is used.
+This can happen when the variable is not initiated, has a negative value in an objective which requires a positive amount or other things.
+
+Important: Nobody will be informed when the default value is used!
+
+Also expect this feature to get changed in the future.
```

Comment:
This need to be on this page, below this entry: https://docs.betonquest.org/2.0-DEV/Documentation/Scripting/About-Scripting/#global-objectives